definitions/variable/tag_fuel_types.yaml

```diff
@@ -74,6 +74,15 @@
   - Coal|Hard coal|w/o CCS:
       description: anthracite and bituminous coal with freely vented CO2 emissions
 
+  - Energy Storage System:
```

Comment:
Could you please remove this change from this PR and start it as a separate discussion? We should probably distinguish between fuels, energy carriers and power-plant types more carefully.

Reply:
Ok, got it.